templates/event.php

```diff
@@ -18,6 +18,9 @@
 
 /* translators: %s: Event title. */
 gp_title( sprintf( __( 'Translation Events - %s' ), esc_html( $event_title ) ) );
+$breadcrumb   = route::get_base_breadcrumb();
```

Comment:
I was confused initially because I didn't realize `route` meant the `Route` class. I think it would be best to capitialize `route` so it matches the class name, `Route`. I'm surprised that it works with a lowercase `r`, and that `phpcs` doesn't complain.

Reply:
No longer relevant since a new approach is being taken now.